api/src/main/java/io/strimzi/api/kafka/model/status/KafkaClusterRebalanceStatus.java

```diff
@@ -41,4 +42,14 @@
     public void setOptimizationResult(Map<String, Object> optimizationResult) {
         this.optimizationResult = optimizationResult;
     }
+
+    @Description("The identification number of the last request sent to Cruise Control")
```

Comment:
Is it a number? I thought it was a uuid? And isn't this really a session identifier between the operator and CC wrt the requests being made for this CR? So I would suggest a description like this:

```suggestion
    @Description("The session identifier for requests to CruiseControl pertaining to this KafkaClusterRebalance resource.")
```

And maybe the property should just be `sessionId`, or something?

Reply:
Yes "number" is not the right word to use, it's a uuid. About naming I was using the way it's named in CC but I am fine with sessionId as well.